config/http.conf

```diff
@@ -48,6 +63,9 @@ http {
   proxy_set_header    Connection          "";
   proxy_set_header    Host                $host;
 
+  # Mitigate httpoxy vulnerability
+  proxy_set_header    Proxy               "";
```

Comment:
Mitigates [httpoxy vulnerability](https://www.nginx.com/blog/mitigating-the-httpoxy-vulnerability-with-nginx/).